bpf/enhancedrecording/network.bpf.c

```diff
@@ -8,6 +8,9 @@
 
 char LICENSE[] SEC("license") = "Dual BSD/GPL";
 
+// Global toggle for UDP tracing.
+u8 udp_disable SEC(".data") = 1;
```

Comment:
I've gone with disabled by default for the BPF C module (to be extra safe), but enabled by default for the feature itself. Seems like a reasonable choice to me, although we can make the entire feature disabled if that seems safer.

FYI @klizhentas @zmb3